spec/components/devolved_nations_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+describe "Devolved Nations", type: :view do
+  def component_name
+    "devolved_nations"
+  end
+
+  it "renders a devolved nations component correctly" do
+    render_component(href: "/back-me")
+    assert_select '.govuk-devolved_nations', text: "Back"
+  end
```

Comment:
I think there needs to be more tests here, to cover all the possible configurations of the component.

Reply:
Yep i've now added 4 different tests which should cover the main use cases (based on the examples I've added in the component guide).